components/TopBarProfileMenu.js

```diff
@@ -38,12 +41,69 @@ const HideGlobalScroll = createGlobalStyle`
 `;
 
 const UserMenuLinkEntry = props => {
+  const { profileMenuLink } = props;
   return (
-    <ListItem mb="6px">
+    <ListItem mb="6px" py={profileMenuLink ? '2' : '0'}>
       <StyledLink as={Link} fontWeight="500" fontSize="14px" lineHeight="20px" color="black.800" {...props} />
     </ListItem>
   );
 };
+UserMenuLinkEntry.propTypes = {
+  profileMenuLink: PropTypes.bool,
+};
+
+const MyAccount = props => {
+  const { LoggedInUser, logout } = props;
+
+  return (
+    <Box as="ul" p={3} my={2}>
+      <Query
+        query={memberInvitationsCountQuery}
+        variables={{ memberCollectiveId: LoggedInUser.CollectiveId }}
+        fetchPolicy="network-only"
+      >
+        {({ data, loading }) =>
+          loading === false && data && data.memberInvitations && data.memberInvitations.length > 0 ? (
+            <UserMenuLinkEntry profileMenuLink href="/member-invitations">
+              <FormattedMessage
+                id="menu.pendingInvitations"
+                defaultMessage="Pending Invitations ({numberOfInvitations})"
+                values={{ numberOfInvitations: data.memberInvitations.length }}
+              />
+            </UserMenuLinkEntry>
+          ) : null
+        }
+      </Query>
+      <UserMenuLinkEntry profileMenuLink href={`/${LoggedInUser.collective.slug}/edit`}>
+        <FormattedMessage id="Settings" defaultMessage="Settings" />
+      </UserMenuLinkEntry>
+      <UserMenuLinkEntry profileMenuLink href={`/${LoggedInUser.username}/recurring-contributions`}>
+        <FormattedMessage id="menu.subscriptions" defaultMessage="Manage Contributions" />
+      </UserMenuLinkEntry>
+      <UserMenuLinkEntry profileMenuLink href={`/${LoggedInUser.username}/transactions`}>
+        <FormattedMessage id="menu.transactions" defaultMessage="Transactions" />
+      </UserMenuLinkEntry>
+      <UserMenuLinkEntry profileMenuLink href="/applications">
+        <FormattedMessage id="menu.applications" defaultMessage="Applications" />
+      </UserMenuLinkEntry>
+      <UserMenuLinkEntry profileMenuLink as="a" href="https://docs.opencollective.com">
+        <FormattedMessage id="menu.help" defaultMessage="Help" />
+      </UserMenuLinkEntry>
+      <UserMenuLinkEntry profileMenuLink as="a" data-cy="logout" onClick={logout}>
+        <Flex alignItems="center">
+          <P color="#7A0F2B" fontWeight="500" pr={2} whiteSpace="nowrap">
```

Comment:
Note that we already have this color at, https://github.com/opencollective/opencollective-frontend/blob/9b33639bfad126a681d5561dfce4127456f25371/lib/theme/colors.js#L80. It was pointed out to me recently that (thanks @Betree 😄) that in Figma designs we mention the color weight next to it as a comment; notice that in this figma design there's a comment 800 marked next to it. 😄 

```suggestion
          <P color="red.800" fontWeight="500" pr={2} whiteSpace="nowrap">
```